guides/common/modules/con_components-of-a-load-balanced-setup.adoc

```diff
@@ -0,0 +1,23 @@
+[id="components-of-a-load-balanced-setup_{context}"]
+= Components of a load-balanced setup
```

Comment:
```suggestion
= Load balancing architecture
```

If the module described each "component" in detail, I would use the current title. But it is higher level, so I think it's more of an architecture description.

Reply:
The issue of using the term `architecture` for anything else than actual `system architecture` came up during a PR a few months ago. Additionally, I don't agree that a module needs to describe components (or anything else, really) in detail to justify using the word in headings: the module _is_ about components and their interactions.

Reply:
Then maybe we only need to call this "Components". :-)